app/interactors/create_user.rb

```diff
@@ -0,0 +1,25 @@
+class CreateUser
+  include Interactor
+
+  def call
+    validate_input
+    execute
+    validate_output
+  end
+
+  private
+
+  def validate_input
+    context.fail!(error: "invalid user params") unless context.user_params
+  end
+
+  def execute
+    context.user = User.create!(context.user_params)
+  rescue
```

Comment:
@Enriikke perhaps I should specify which type of error I'm rescuing from? Is it bad practice to just have a blanket `rescue`?